Fixed instanceof summary to better match instanceof page (#35062)
mdn/content: files/en-us/web/javascript/guide/expressions_and_operators/index.md

if the specified object is of the specified object type. The syntax is:

```js-nolint
object instanceof objectType
```

> where `object` is the object to test against `objectType`, and `objectType` is an object type, such as {{jsxref("Date")}} or {{jsxref("Array")}}.

Reviewer: ```suggestion
where `object` is the object to test against `objectType`, and `objectType` is a constructor, such as {{jsxref("Date")}} or {{jsxref("Array")}}.
```
